data/category-ntp

```diff
@@ -2,16 +2,16 @@
 regexp:^time(.euro)?.apple.com$
 
 # Facebook
-regexp:^time([1-5])?.facebook.com$
+regexp:^time([1-5])?\.facebook\.com$
 
 # Google
 regexp:^time([1-4])?.google.com$
 
 # NIST: https://tf.nist.gov/tf-cgi/servers.cgi
-regexp:^((((ut1-wwv)|(ntp\-(b|d|wwv))|(time(\-(([a-e]\-(g|b))|([a-e]\-wwv)))?)).nist.gov)|((ntp-c|ut1-time)|(utcnist(2|3)?)).colorado.edu)$
+^((((ut1-wwv)|(ntp\-(b|d|wwv))|(time(\-(([a-e]\-(g|b))|([a-e]\-wwv)))?))\.nist\.gov)|((ntp-c|ut1-time)|(utcnist(2|3)?))\.colorado\.edu)$
```

Comment:
The `regexp:` prefix is missing here.

Reply:
I will solve this problem in another PR.